bazelisk.py

```diff
@@ -92,26 +94,36 @@ def resolve_version_label_to_number(bazelisk_directory, version):
 
 
 def determine_bazel_filename(version):
-  machine = platform.machine()
+  machine = normalized_machine_arch_name()
   if machine != 'x86_64':
     raise Exception('Unsupported machine architecture "{}". '
                     'Bazel currently only supports x86_64.'.format(machine))
 
-  operating_system = platform.system().lower()
-  if operating_system not in ('linux', 'darwin', 'windows'):
+  if OPERATING_SYSTEM not in ('linux', 'darwin', 'windows'):
     raise Exception('Unsupported operating system "{}". '
                     'Bazel currently only supports Linux, macOS and Windows.'
-                    .format(operating_system))
+                    .format(OPERATING_SYSTEM))
+
+  if machine == 'amd64':
```

Comment:
I think we no longer need this, because you pulled out this normalizing logic into the `normalized_machine_arch_name` method?

Reply:
Oops, indeed! I'm removing this locally.

Reply:
Done.